exporter/dynatraceexporter/README.md

```diff
@@ -37,6 +41,9 @@ receivers:
 
 processors:
   batch:
+    # Batch size must be less than or equal to 1000
+    send_batch_max_size: 1000
```

Comment:
```suggestion
    send_batch_size: 1000
```

According to this, `send_batch_max_size` is only supported for the trace pipeline:
https://github.com/open-telemetry/opentelemetry-collector/blob/68c5961f7bc2d8d55fd431ab07367fc16f8b26d0/processor/batchprocessor/README.md#L18-L25

Reply:
gah you're right. Looks like it was introduced here https://github.com/open-telemetry/opentelemetry-collector/pull/1310/files and is still enforced that way

Reply:
obsolete, metrics support added in https://github.com/open-telemetry/opentelemetry-collector/pull/2422